lib/process-services-cloud/src/lib/process/process-filters/components/edit-process-filter-cloud.component.spec.ts

```diff
@@ -211,112 +210,19 @@ describe('EditProcessFilterCloudComponent', () => {
-                options[2].nativeElement.click();
-                fixture.detectChanges();
-                fixture.whenStable().then(() => {
-                    const saveButton = fixture.debugElement.nativeElement.querySelector('[data-automation-id="adf-filter-action-save"]');
-                    expect(saveButton.disabled).toEqual(false);
-                });
-            }));
-
-            it('should disable save button if the filter is not changed for custom filter', async(() => {
-                component.toggleFilterActions = true;
-                const expansionPanel = fixture.debugElement.nativeElement.querySelector('mat-expansion-panel-header');
-                expansionPanel.click();
-                fixture.detectChanges();
-                fixture.whenStable().then(() => {
-                    const saveButton = fixture.debugElement.nativeElement.querySelector('[data-automation-id="adf-filter-action-save"]');
-                    expect(saveButton.disabled).toBe(true);
-                });
-            }));
         });
 
         describe('SaveAs Button', () => {
-
-            it('should disable saveAs button if the process filter is not changed for default filter', async(() => {
-                getProcessFilterByIdSpy.and.returnValue(of({
-                    id: 'filter-id',
-                    name: 'ADF_CLOUD_PROCESS_FILTERS.RUNNING_PROCESSES',
-                    sort: 'my-custom-sort',
-                    processDefinitionId: 'process-definition-id',
-                    priority: '12'
-                }));
-
-                const processFilterIdChange = new SimpleChange(null, 'filter-id', true);
-                component.ngOnChanges({ 'id': processFilterIdChange });
-                fixture.detectChanges();
-
-                component.toggleFilterActions = true;
-                const expansionPanel = fixture.debugElement.nativeElement.querySelector('mat-expansion-panel-header');
-                expansionPanel.click();
-                fixture.detectChanges();
-                fixture.whenStable().then(() => {
-                    const saveButton = fixture.debugElement.nativeElement.querySelector('[data-automation-id="adf-filter-action-saveAs"]');
-                    expect(saveButton.disabled).toEqual(true);
-                });
-            }));
-
-            it('should disable saveAs button if the process filter is not changed for custom filter', async(() => {
-                component.toggleFilterActions = true;
-                const expansionPanel = fixture.debugElement.nativeElement.querySelector('mat-expansion-panel-header');
-                expansionPanel.click();
-                fixture.detectChanges();
-                fixture.whenStable().then(() => {
-                    const saveButton = fixture.debugElement.nativeElement.querySelector('[data-automation-id="adf-filter-action-saveAs"]');
-                    expect(saveButton.disabled).toEqual(true);
-                });
-            }));
```

Comment:
The unit tests that were removed were redundant/duplicated/not needed/covered by e2e?

Reply:
Not needed or redundant, yes. For example, "save as" should not depend on the dirty state of the form, I remove that linking and some tests for that become irrelevant. Same for other areas. There are still a few useless tests in the e2e (or duplicates), but I will address that later I hope.

Reply:
Thnx 👍